components/CancellationList.vue

```diff
@@ -16,38 +16,38 @@
       loading-text="Loading... Please wait"
       :search="search"
     >
-      <template v-slot:[`item.name`]="{ item }" align="left">
+      <template #[`item.name`]="{ item }" align="left">
         <div class="dflex justify-left">
           {{ item.name }}
         </div>
       </template>
 
-      <template v-slot:[`item.note`]="{ item }" align="left">
+      <template #[`item.note`]="{ item }" align="left">
         <div
           v-if="item.note && item.note.length > 1"
           class="dflex justify-center"
         >
-          <v-icon @click="showNote(item.note)" color="cyan darken-1" center
-            >mdi-note-text</v-icon
-          >
+          <v-icon color="cyan darken-1" center @click="showNote(item.note)">
```

Comment:
It has some opinions on ordering things. Just give me a 👍🏻 or 👎🏻 depending on how you feel on these 😆